Module.yaml

```diff
@@ -28,6 +28,27 @@ environments:
     default: 80
     type: integer
     name: Port
+  - key: AUTHENTICATION_REQUIRED
+    description: Connection requires authentication via bearer token
+    value: ''
+    default: no
+    type: enum
+    options:
+      - yes
+      - no
+    name: Authentiocation Bearer   
+  - key: ACCESS_TOKEN
+    description: Authentication token for authorization
+    value: ''
+    default: ''
+    type: string
+    name: Authentication token
+  - key: AUTHENTICATION_API_KEY
+    description: Authentication API key if required
+    value: ''
+    default: ''
+    type: string
+    name: Authentication API key
```

Comment:
We don't need this changes. Please, revert it.

Reply:
done.